charts/ai-software-templates/chatbot/README.md.gotmpl

```diff
@@ -10,7 +10,12 @@
 This Helm Chart creates the following components:
 
 ### The Model Service
-Based on the `llama.cpp` inference server and related to the [ai-lab-recipes model server](https://github.com/containers/ai-lab-recipes/tree/main/model_servers/llamacpp_python).
+By default the `chatbot-ai-sample` supports the `llama.cpp` inference server and related to the [ai-lab-recipes model server](https://github.com/containers/ai-lab-recipes/tree/main/model_servers/llamacpp_python).
```

Comment:
can you replicate my suggestions for this from the README.md file

Reply:
Fixed in [87d9c1e](https://github.com/redhat-ai-dev/ai-lab-helm-charts/pull/33/commits/87d9c1eb6630c67aa40b4c29ed17014d321a90d5)